src/ray/gcs/gcs_server/gcs_task_manager.h

```diff
@@ -168,12 +238,17 @@ class GcsTaskManager : public rpc::TaskInfoHandler {
     absl::flat_hash_map<TaskAttempt, size_t> task_attempt_index_;
 
     /// Secondary index from task id to task attempts.
+    /// TODO(rickyx): use unordered_multimap for these as well.
```

Comment:
Thinking about it again - i don't think btree would be better than a hash map here since it would take O(logn) to traverse the tree and we don't really need the sorting property.

Reply:
sycned offline